extensions/amp-ad-network-adsense-impl/0.1/amp-ad-network-adsense-impl.js

```diff
@@ -165,6 +187,40 @@ export class AmpAdNetworkAdsenseImpl extends AmpA4A {
       addExperimentIdToElement(verifierEid, this.element);
     }
 
+    const correlator = getCorrelator(this.win);
+    this.ampAnalyticsPageLoadMetricsConfig_ = dict({
+      'requests': {
+        'fvt': 'https://csi.gstatic.com/csi?s=a4a' +
+            `&c=${correlator}` +
+            '&firstVisibleTime.${firstVisibleTime}',
+      },
+      'transport': {
+        'beacon': false,
+        'xhrpost': false,
+      },
+      'triggers': {
+        'iniLoad': {
+          'on': 'visible',
+          'request': 'fvt',
+          'selector': 'amp-ad',
+          'selectionMethod': 'closest',
+        },
+      },
+    });
+    // TODO(jonkeller): Add remaining metrics commented below to config
+    // 'met.a4a=makeBodyVisible.${MBV_VALUE}~' +
+    // 'firstContentfulPaint.${FCP_VALUE}~' +
+    // 'firstViewportReady.${FVR_VALUE}',
+    if (this.ampAnalyticsPageLoadMetricsConfig_) {
```

Comment:
Didn't you just set this above?  Is there any way this `if` could ever fail?

Reply:
You're right, there's no way it could. Fixed.